### DISCUSSION TITLE
Rapidly changing changelog listings cause a crash while disconnected from the internet

### DISCUSSION THREAD

**Discussion by TheRealStevie** [ppy/osu#24474] [Q&A] [answered]
If you're disconnected from the internet (done manually at 0:16 in the video), then by changing the changelog listings quick enough can cause a crash.

https://github.com/ppy/osu/assets/75444413/12b4cfc0-154d-47c1-a948-ca22ce2ffb8c

[database.log](https://github.com/ppy/osu/files/12269105/database.log)
[input.log](https://github.com/ppy/osu/files/12269106/input.log)
[network.log](https://github.com/ppy/osu/files/12269107/network.log)
[performance.log](https://github.com/ppy/osu/files/12269108/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12269109/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12269110/updater.log)

Looking at the runtime logs, I thought there was something similar to the logs it pointed out at https://github.com/ppy/osu/pull/20504 that fixes [an issue I reported](https://github.com/ppy/osu/discussions/20448), so I'll reference it here just in case.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/24478
